fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/FedoraLdpIT.java

```diff
@@ -421,6 +424,179 @@ public void testHeadDatastreamWithWantDigestMultiple() throws IOException, Parse
         }
     }
 
+    @Test
+    public void testExternalDatastreamProxyWithWantDigestForLocalFile() throws IOException, ParseException {
+
+        final File externalFile = createExternalLocalFile("01234567890123456789012345678901234567890123456789");
```

Comment:
I would suggest creating constants for the file content string as well as the sha and md5 values to reduce the duplication in all the tests where these values appear.